src/controllers/transaction.controller.js

```diff
@@ -7,12 +7,14 @@ export default {
   * @param {string} res
   * @returns {object} returns an array of all transactions
   */
-  index(req, res) {
-    const transactions = TransactionModel.all();
-
-    if (transactions) { return res.json({ status: 200, transactions }); }
-
-    return res.json({ status: 400, message: 'No transation available' });
+  async index(req, res) {
```

Comment:
rename this function, `index` is not descriptive enough

Reply:
Noted.